### PR TITLE
chore: update module

### PR DESCRIPTION
**Pull request by ricardogobbosouza**
- Increase coverage
- Update badges
- Add commitlint
- Add husky

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,13 +12,13 @@
     }
   ],
   "main": "lib/module.js",
-  "repository": "nuxt-community/vuetify-module",
+  "repository": "https://github.com/nuxt-community/vuetify-module",
```
Comment (pi0) [inline package.json]: This is not necessary

File: test/module.test.js [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
 describe('module', () => {
   let nuxt
 
   beforeAll(async () => {
-    nuxt = new Nuxt(require('./fixture/nuxt.config'))
+    nuxt = new Nuxt(config)
     await nuxt.ready()
     await new Builder(nuxt).build()
-  }, 60000)
+  })
```
Comment (pi0) [inline test/module.test.js]: Timeout can be passed as second argument here

File: test/module.test.js [hunk 3]
```diff
@@ -18,3 +23,30 @@ describe('module', () => {
...
   })
 })
+
+describe('disable all default assets', () => {
+  let nuxt
+
+  beforeAll(async () => {
+    nuxt = new Nuxt({
+      ...config,
+      ...{
```
Comment (pi0) [inline test/module.test.js]: Can be simplified to:

```js
new Nuxt({
   ...config,
   vuetify: {
        defaultAssets: false
   }
}
```

File: test/module.test.js (hunk 2 above)
Comment (ricardogobbosouza) [reply test/module.test.js]: This is being used `jest.setTimeout(60000)`

Comment (pi0) [reply test/module.test.js]: Is it in latest template?

Comment (pi0) [reply test/module.test.js]: (If so, we have to also update it. 60sec is too much and only required for build)

Comment (ricardogobbosouza) [reply test/module.test.js]: Yes, it is. But in the template build and listen

**Comment by galvez** [comment]
@pi0 @aldarund should we bump to v2.0.0-beta.8 too?

**Comment by aldarund** [comment]
ye, @galvez maybe u have idea why fibers fails?

**Comment by kevinmarrec** [comment]
@ricardogobbosouza Is this PR ready to be merged ?

**Comment by ricardogobbosouza** [comment]
Hi @kevinmarrec 
Skipped test for now
I will resolve the conflicts and it's ready to be merged

**Comment by ricardogobbosouza** [comment]
@kevinmarrec it's ready

**Comment by kevinmarrec** [comment]
Thanks @ricardogobbosouza !

**Comment by kevinmarrec** [comment]
@ricardogobbosouza Tests seem to be passing but silently output a bunch of errors related to `vuetify-loader` :
Screenshot from 2019-07-22 17-13-47

Any ideas ?

**Comment by ricardogobbosouza** [comment]
@kevinmarrec 
It is this test https://github.com/nuxt-community/vuetify-module/pull/48/commits/da0a868bc98e2676c1df3c5d74eae615988be92e
That's why it was skipped

**Comment by kevinmarrec** [comment]
Why is it not skipped locally ? I mean it shouldn't be runned when I do `yarn test`, right ?

**Comment by ricardogobbosouza** [comment]
Exactly, should not run, nor launch error silently

**Comment by ricardogobbosouza** [comment]
Exactly, should not run, nor launch error silently

**Pull request merged**